cmd/node/node.go

```diff
@@ -348,6 +352,25 @@ func (app SpacemeshApp) stopServices() {
 
 }
 
+func getEdIdentity() *signing.EdSigner {
+	// read key from file if exist
+	log.Info("Reading identity from file")
+	dataDir, err := filesystem.GetSpacemeshDataDirectoryPath()
+	if err != nil {
+		log.Error("Could not get data path err=%v", err)
+		return signing.NewEdSigner()
+	}
+
+	buff, e1 := ioutil.ReadFile( dataDir + "/identity.ed")
```

Comment:
the name of the file should be a const field and not hardcoded as it is now

Reply:
What do you mean by "a const field"?